shopinvader/services/cart.py

```diff
@@ -321,8 +379,9 @@ def _check_existing_cart_item(self, cart, params):
         )
 
     def _prepare_cart_item(self, params, cart):
+        product = self.env["product.product"].browse(params["product_id"])
```

Comment:
@acsonefho Why `browse` here

Reply:
@lmignon Completely useless! It was a part of my previous "try to fix" that I forgot to remove (I just remove it)